microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Edit files with Workspaces (Filesystem tab)
-description: How to save changes made within DevTools to disk.
+title: Edit webpage files with a DevTools workspace
```

Comment:
Consider 'using' instead of 'with' per corp style guide
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with
or some wording other than 'with'.  eg:
Edit webpage files by using a Workspace (Filesystem tab)

When changing a title, update the string in all of the usual spots:
toc
line 2 "title:"
h1
Link text of all links that point to here.

Reply:
will do